examples/primitive_types/src/primitive_types.ts

```diff
@@ -132,4 +133,15 @@ export function getPrincipal(): Query<Principal> {
 export function printPrincipal(principal: Principal): Query<Principal> {
     ic.print(typeof principal);
     return principal;
-}
\ No newline at end of file
+}
+
+export function getEmpty(): Query<empty> {
+    throw 'Anything you want';
+}
+
+// Note: It is impossible to get into this function because it requires an
```

Comment:
I suggest the word `call` instead of `get into` and `pass an "empty" value as an argument` instead of `pass an "empty" value.`